docs/spec_draft.md

```diff
@@ -242,10 +244,9 @@ Produces a `result` tensor from a constant `value`.
 ### Semantics
 
 Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
-`result` tensor. For floating-point element type, implements IEEE 754 semantics:
-Returns the larger of two operands, propagating `NaN`s and treating `-0` as
-less than `+0`. For complex element type,  performs lexicographic comparison on
-the (real, imaginary) pairs.
+`result` tensor. For floating-point element types, implements the maxNum
```

Comment:
Per https://en.wikipedia.org/wiki/IEEE_754, "As of 2019, the formerly required minNum, maxNum, minNumMag, and maxNumMag in IEEE 754-2008 are now deleted due to their non-associativity. Instead, two sets of new minimum and maximum operations are recommended... The first set contains minimum, minimumNumber, maximum and maximumNumber. The second set contains minimumMagnitude, minimumMagnitudeNumber, maximumMagnitude and maximumMagnitudeNumber. The history and motivation for this change are explained in a background document".

Should we use `maximum` or `maximumNumber` here? (Same question for `minimum` and `minimumNumber` below).

Reply:
Thanks for the information!

IMO, we should go with the `maximum`/`minimum` (with NaN propagation) as that is the same behavior displayed by the corresponding HLO operation.

Reply:
Following the behavior of the corresponding HLO operation SGTM for the spec draft.